pages.id/sunos/snoop.md

```diff
@@ -1,7 +1,7 @@
 # snoop
 
 > Pengendus paket jaringan.
-> tcpdump untuk SunOS.
+> SunOS setara dengan tcpdump.
```

Comment:
No, this significantly change the context from `the SunOS equivalent to the tcpdump command` to become `this SunOS platform is the same as tcpdump`.

I think it’s okay to use the current translation.

Reply:
Aren't we ignoring this rule here/restricting it to English pages? 

Maybe we can update tldr-maintenance to do the same i.e. only check this for English pages.

Reply:
> > I think it’s okay to use the current translation.
> 
> It isn't since that is causing a linter issue. Could you rephrase the sentence in a way that it starts with a capital letter?

Well it could be

```suggestion
Perintah setara tcpdump untuk SunOS.
```